raiden_contracts/deploy/__main__.py

```diff
@@ -208,6 +208,12 @@ def raiden(
     type=int,
     help="Maximum amount of tokens deposited in UserDeposit",
 )
+@click.option(
+    "--initial_service_deposit",
```

Comment:
I think `initial_service_price` would fit better

Reply:
Now I hear it, I agree.   Launching `raiden deploy ... --initial_service_deposit 30000` sounds like you're already making a deposit, but actually you're only setting a price.